src/applications/components/memorialTextarea/style.ts

```
@@ -49,3 +48,11 @@ export const CommitArea = styled(TextareaAutosize)`
   outline: none;
   min-height: 50px;
 `;
+export const Set = styled.div`
```

Biome (1.9.4)

[error] Do not shadow the global "Set" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)